### ISSUE TITLE
What's with all the resolutions?

### ISSUE THREAD

**Issue by jordwalke** [osener/bonsai_revery#1]
Hi I just noticed that the package.json has a ton of resolutions.
Resolutions tell the constraint solver "I know better than you!" which is sometimes true, but adding resolutions also has some downsides. I think of them as like a stop gap when one of your dependencies doesn't model their dependencies correctly and you need to force the solver to solve it a certain way while you wait for them to fix it.
Also, resolutions during development can mask your own project's incorrectly configured constraints because resolutions tell the constraint solver to ignore your own dependencies specified versions. So then when someone tries out your library, the constraints you specified in `"dependencies"` won't work.

I know everyone has their own workflow, and whatever works for you is best - just thought I'd bring up the point in case you hadn't considered it.

**Comment by ozanvos** [comment]
Hi @jordwalke. I see what you mean, resolutions definitely make this library harder to use. I wanted to base this off of Bonsai's unreleased development version `v0.14` because of some major backwards incompatible changes. This version depends on rest of Jane Street libraries being on ` v0.14` as well. 

I tried using Jane Street's "bleeding" repository with Esy but struggled to get it going because it doesn't support multiple opam repositories (`janestreet-bleeding` only contains Jane Street packages), and even if I did it wouldn't have made it easy to use bonsai_revery anyway.
